cryptoservice/import_export.go

```diff
@@ -99,18 +104,45 @@ func (cs *CryptoService) ExportRootKeyReencrypt(dest io.Writer, keyID string, ne
 // It prompts for the key's passphrase to verify the data and to determine
 // the key ID.
 func (cs *CryptoService) ImportRootKey(source io.Reader) error {
+	return cs.ImportRoleKey(source, data.CanonicalRootRole, nil)
+}
+
+// ImportRoleKey imports a private key in PEM format key from an io.Reader
+// It prompts for the key's passphrase to verify the data and to determine
+// the key ID.
+func (cs *CryptoService) ImportRoleKey(source io.Reader, role string, newPassphraseRetriever passphrase.Retriever) error {
```

Comment:
Should `ImportRootKey` just be changed to `ImportRoleKey` everywhere?  Since currently, `ImportRootKey` is in the interface, but `ImportRoleKey` is not.